lib/pengine/tests/rules/pe_cron_range_satisfied_test.c

```diff
@@ -11,7 +11,7 @@ run_one_test(const char *t, const char *x, int expected) {
     g_assert_cmpint(pe_cron_range_satisfied(tm, xml), ==, expected);
 
     crm_time_free(tm);
-    xmlFreeNode(xml);
+    xmlFreeDoc(xml->doc);
```

Comment:
this is fine, but fyi free_xml() handles this generally

we should add a copyright/license notice to this file